### PR TITLE
data:Mvishal123

### PR DESCRIPTION
**Pull request by Mvishal123**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2773"></a>

### CODE REVIEW

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please remove the comma on line 40 & the blank line 41

**Review by dan-mba — APPROVED** [review]

**Comment by Mvishal123** [comment]
Hey I have removed the  extra line and the comma

**Review by dan-mba — APPROVED** [review]
Looks good to me

File: data/Mvishal123.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Vishal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-source | DSA | Blog | Freelance | Loading..",
+  "avatar": "https://twitter.com/Mvishalx.png",
```
Comment (kumarsonsoff3) [inline data/Mvishal123.json:6]: Please fix the broken avatar link. Twitter doesn't allow you to use pfp link in this way! Instead, it's better to use the default one. 🛠

```suggestion
  "avatar": "https://github.com/Mvishal123.png",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Mvishal123.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/Mvishalx",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Check out my Blogs here",
+      "url": "https//mvishal.hashnode.dev/",
```
Comment (kumarsonsoff3) [inline data/Mvishal123.json:32]: Please fix the broken blog URL. 🛠

```suggestion
      "url": "https://mvishal.hashnode.dev/",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by Mvishal123** [comment]
I have fixed the broken links

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 👍

**Pull request merged**